flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/ConfigVariablesManager.java

```diff
@@ -47,6 +51,8 @@
  */
 public class ConfigVariablesManager {
 
+    public static final String DEFAULTABLE = "defaultable";
+    public static final String LS = System.lineSeparator();
```

Comment:
More intuitive to use 

```suggestion
    public static final String LINE_SEPATATOR = System.lineSeparator();
```